File: src/main/scala/de/mineformers/allomancy/internal/network/EntityMetalStorageUpdate.scala
package de.mineformers.allomancy.internal.network

import de.mineformers.allomancy.metal.MetalStorage
import tessera.io.Message

/**
  * EntityMetalStorageUpdate
  *
  * @author PaleoCrafter
  */
case class EntityMetalStorageUpdate(var entity: Int, var storage: MetalStorage) extends Message {
  def this() = this(-1, null)
}

File: src/main/java/de/mineformers/allomancy/network/messages/EntityMetalStorageUpdate.java
package de.mineformers.allomancy.network.messages;

import de.mineformers.allomancy.metal.MetalStorage;
import de.mineformers.allomancy.network.Message;

/**
 * EntityMetalStorageUpdate
 *
 * @author PaleoCrafter
 */
public class EntityMetalStorageUpdate extends Message {
    public int entity;
    public MetalStorage storage;

    public EntityMetalStorageUpdate() {
    }

    public EntityMetalStorageUpdate(int entity, MetalStorage storage) {
        this.entity = entity;
        this.storage = storage;
    }
}
